### PR TITLE
Add set_as_template to meeting.{create,update}

### LINKED ISSUE #1214: Update the actions `meeting.create` and `meeting.update`
I have noticed, that the actions `meeting.create` and `meeting.update` do not expect the key `set_as_template` (correct me, if I'm wrong).
However, I suggest to rename it `template_for_committee_id`, so it is easier to pass and receive this field to and from the backend.
So, if you agree with me, then please update the actions to match the description in the wiki:
- [x] [meeting.create](https://github.com/OpenSlides/OpenSlides/wiki/meeting.create)
- [x] [meeting.update](https://github.com/OpenSlides/OpenSlides/wiki/meeting.update)

### PR DESCRIPTION
**Pull request by jsangmeister**
closes #1214

### CODE REVIEW

**Review by r-peschke — COMMENTED** [review]
Technically I would approve the changes. But ...

### We should not allow a template-meeting to be used as regular meeting.
A template meeting should not be changed by daily work on motions, polls etc.

### Requirements and database structure

case 1:
Requirement: Each committee can define template_meetings, which are available for the whole organization
With this requirement we have to read all committees with the field `template_meeting_ids`, to present all organization wide meeting-templates.

case 2:
Requirement: Each committee can define meeting-templates only for themselves.   
The design fits best for this case

case 3:
Requirement: More flexibility with meeting-templates for the whole organization or the committee itself or a dedicated list of committees. There are some combinations possible. 
For this case we should define the exact requirement and create a database structure.

**Comment by jsangmeister** [comment]
> ### We should not allow a template-meeting to be used as regular meeting.
> A template meeting should not be changed by daily work on motions, polls etc.

How do you want to prevent this? The user still needs to be able to edit the template and everything in it (motions, agenda, etc.), so it must be usable just like a "real" meeting. I think it is the user's responsibility to not use a template meeting as a "regular" one. Only thing we could do is display a warning, but this is would be a client change...

**Review by r-peschke — CHANGES_REQUESTED** [review]

**Comment by r-peschke** [comment]
> > ### We should not allow a template-meeting to be used as regular meeting.
> > A template meeting should not be changed by daily work on motions, polls etc.
> 
> How do you want to prevent this? The user still needs to be able to edit the template and everything in it (motions, agenda, etc.), so it must be usable just like a "real" meeting. I think it is the user's responsibility to not use a template meeting as a "regular" one. Only thing we could do is display a warning, but this is would be a client change...

This is the user's responsibility! He want's and can change the template-meeting.

File: global/data/example-data.json [hunk 1]
```diff
@@ -516,7 +517,6 @@
             "meeting_ids": [
                 1
             ],
-            "template_meeting_ids": [],
```
[the remark this answers is not in the document]
Comment (jsangmeister) [reply global/data/example-data.json:519]: No, since there are no templates currently in use, this should be fine

**Comment by ostcar** [comment]
See https://github.com/OpenSlides/openslides-autoupdate-service/pull/423 for the autoupdate-service.

Please ping me, if the models.yml is changed again in this PR

**Comment by jsangmeister** [comment]
Added migration. Only works with https://github.com/OpenSlides/openslides-datastore-service/pull/180

**Review by r-peschke — APPROVED** [review]

**Pull request merged**